resources.go

```diff
@@ -169,11 +169,12 @@ func Provider() tfbridge.ProviderInfo {
 			"packet_operating_system":    {Tok: makeDataSource(mainMod, "getOperatingSystem")},
 			"packet_precreated_ip_block": {Tok: makeDataSource(mainMod, "getPrecreatedIpBlock")},
 			"packet_spot_market_price":   {Tok: makeDataSource(mainMod, "getSpotMarketPrice")},
+			"packet_device":              {Tok: makeDataSource(mainMod, "getDevice")},
```

Comment:
Bugfix?

Reply:
Yes, this was missing from the SDK generation and I found it now and added it in

Reply:
kk, might add that to the CL

Reply:
👍